cpp/src/aztec3/utils/types/native_types.hpp

```diff
@@ -44,6 +44,11 @@ struct NativeTypes {
     // typedef barretenberg::g1::element bn254_jac_point;
     // typedef barretenberg::g1 bn254_group;
 
+    typedef secp256k1::g1 secp256k1_group;
```

Comment:
FYI: Some of these types are not declared in `circuit_types.hpp` because we can access them from the `secp256k1<Composer>` struct anyway. For example,
`NT::secp256k1_fr ≡ CT::secp256k1::Fr`.

Reply:
How do we access the _circuit type_ (field_t version) of `CT::secp256k1, if `CT::secp256k1::Fr` gives us a native type?
It seems like specifically declaring `secp256k1_fr` in circuit_types.hpp might help?

Also, using `typedef NCT::secp256k1_fr` doesn't work unless `secp256k1_fr` is defined in both files?